### PR TITLE
Nbo/handle errors at login

### PR DESCRIPTION
**Pull request by nborde-CSM**
Dev Review rules:
- 3 PR max
- at the second feedback, prefer meet instead of github conversation

Definition of Done:
- [ ] Testing
- [ ] No error logs in web console (except iFrame errors)
- [ ] Create or Update documentation (README file for users/integrators)
- [ ] Identify breaking changes with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)
- [ ] Check the [test page](https://spaceport.cosmotech.com/confluence/pages/viewpage.action?spaceKey=QA&title=Azure+Sample+Webapp) in order to manually validate the feature
- [ ] Clean code (regroup configuration variable, typos, no console.log...)

### CODE REVIEW

File: public/locales/fr/translation.json [hunk 1]
```diff
@@ -359,13 +359,17 @@
     "signin": {
       "title": "Exemple d'Application Web Azure"
     },
+    "accessdenied": {
+      "title": "Accès non autorisé",
+      "errormessage": "Les resources n'existent pas ou vous n'avez pas la permission d'y accéder.",
+      "signouttimeout": "Vous allez être déconnecté automatiquement dans {{seconds}}... secondes"
+    },    
     "unauthorized": {
```
Comment (csm-thu) [inline public/locales/fr/translation.json:367]: Can we remove the `unauthorized` block if it is no longer useful ? (or merge it with `accessdenied`)

File: src/views/AccessDenied/AccessDenied.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+              <Typography className={classes.quoteText} variant="h1">
+                {t('views.signin.title', 'Azure Sample Web Application')}
+              </Typography>
+            </div>
+          </div>
+        </Grid>
+        <Grid className={classes.content} item lg={7} xs={12} container>
+          <div className={classes.content}>
+            <div className={classes.contentHeader}>
+              <form className={classes.form}>
```
Comment (csm-thu) [inline src/views/AccessDenied/AccessDenied.js:45]: Do we really need a form here ? There is no `input` to send, we only have a button to force log out

File: src/views/AccessDenied/AccessDenied.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                    renderer={timeoutRenderer}
+                    onComplete={() => Auth.signOut()}
+                  />
+                </Typography>
+                <Button variant="outlined" className={classes.signoutButton} onClick={() => Auth.signOut()}>
+                  {t('genericcomponent.userinfo.button.logout', 'Sign Out')}
+                </Button>
+              </form>
+            </div>
+            <div className={classes.contentBody}></div>
```
Comment (csm-thu) [inline src/views/AccessDenied/AccessDenied.js:67]: Is this div necessary ?

**Review by csm-thu — CHANGES_REQUESTED** [review]

File: src/views/AccessDenied/AccessDenied.js (hunk 2 above)
Comment (nborde-CSM) [reply src/views/AccessDenied/AccessDenied.js:45]: It is a graphic build from sign in page. I have tryed to simplify it but each time it break the design.

File: src/views/AccessDenied/AccessDenied.js (hunk 3 above)
Comment (nborde-CSM) [reply src/views/AccessDenied/AccessDenied.js:67]: Idem

**Pull request merged**